zerver/webhooks/clickup/view.py

```diff
@@ -0,0 +1,223 @@
+        return (
+            "\n~~~ quote\n :silhouette: Unassigned from **{old_value}**\n~~~\n"
+        ).format(old_value=history_data["before"]["username"].tame(check_string))
+    elif updated_field == SPECIAL_FIELDS.COMMENT:
+        return (
+            "\n~~~ quote\n :speaking_head: Commented by **{event_user}**\n~~~\n"
+        ).format(event_user=history_data["user"]["username"].tame(check_string))
+    
+    elif updated_field == SPECIAL_FIELDS.DUE_DATE:
+        old_value = history_data.get("before").tame(check_string) if history_data.get("before") else None
+        old_due_date = convert_unixdatetime(int(old_value), format="%Y-%m-%d") if old_value else None
+        new_value = history_data.get("after").tame(check_string) if history_data.get("after") else None
+        new_due_date = convert_unixdatetime(int(new_value), format="%Y-%m-%d") if new_value else None
+        return (
+            "\n~~~ quote\n :spiral_calendar: Due date updated from **{old_value}** to **{new_value}**\n~~~\n"
+        ).format(old_value=old_due_date,
+                 new_value=new_due_date
+                 )
+    
+    elif updated_field == SPECIAL_FIELDS.MOVED:
+        return (
+            "\n~~~ quote\n :folder: Moved from **{old_value}** to **{new_value}**\n~~~\n"
+        ).format(old_value=history_data["before"]["name"].tame(check_string),
+                 new_value=history_data["after"]["name"].tame(check_string))
+        
+    elif updated_field == SPECIAL_FIELDS.TIME_SPENT:
+        raw_time_spent = history_data.get("data",{}).get("total_time").tame(check_string)
+        new_time_spent =  convert_unixdatetime(int(raw_time_spent), format="%H:%M:%S")
+        return (
+            "\n~~~ quote\n :stopwatch: Time spent changed to **{new_value}**\n~~~\n"
+        ).format(new_value = new_time_spent)
+
+def convert_unixdatetime(unixdatetime: int, format: str) -> str:
+    unixdatetime = unixdatetime / 1000
+    return datetime.fromtimestamp(unixdatetime).strftime(format)
+
+CREATED = "Created"
+UPDATED = "Updated"
+DELETED = "Deleted"
+
+def topic_and_body(payload: WildValue, clickup_api_key: str, team_id: str) -> Tuple[str, str]:
+    event_code = payload["event"].tame(check_string)
+    topic = "ClickUp Notification"
+    
+    event_item_type = event_code.replace(CREATED, "").replace(UPDATED, "").replace(DELETED, "")
+
+    if DELETED in event_code:
+        body = generate_delete_event_message(event_item_type=event_item_type)
+        return topic, body
+
+    item_data = get_item_data(event_item_type=event_item_type, api_key=clickup_api_key, payload=payload, team_id=team_id)
+    print("ITME DATA: ", item_data)
+    if CREATED in event_code:
+        body = generate_create_event_message(item_data=item_data, event_item_type=event_item_type)
+    elif UPDATED in event_code:
+        body = generate_updated_event_message(item_data=item_data,payload=payload, event_item_type=event_item_type)
+
+    return topic, body
```

Comment:
Your editor seems to be misconfigured -- none of these files seem to have newlines at their ends.